packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_webview_widget.dart

```diff
@@ -161,6 +164,112 @@ class WebKitWebViewPlatformController extends WebViewPlatformController {
     };
   }
 
+  @override
+  Future<void> loadHtmlString(String html, {String? baseUrl}) {
+    return webView.loadHtmlString(html, baseUrl);
+  }
+
+  @override
+  Future<void> loadFile(String absoluteFilePath) async {
+    await webView.loadFileUrl(
+      absoluteFilePath,
+      path.dirname(absoluteFilePath),
+    );
+  }
+
+  @override
+  Future<void> loadFlutterAsset(String key) async {
+    assert(key.isNotEmpty);
+    return webView.loadFlutterAsset(key);
+  }
+
+  @override
+  Future<void> loadUrl(String url, Map<String, String>? headers) async {
+    final NSUrlRequest request = NSUrlRequest(
+      url: url,
+      allHttpHeaderFields: headers ?? <String, String>{},
+    );
+    return webView.loadRequest(request);
+  }
+
+  @override
+  Future<void> loadRequest(WebViewRequest request) async {
+    if (!request.uri.hasScheme) {
+      throw ArgumentError('WebViewRequest#uri is required to have a scheme.');
+    }
+
+    final NSUrlRequest urlRequest = NSUrlRequest(
+      url: request.uri.toString(),
+      allHttpHeaderFields: request.headers,
+      httpMethod: describeEnum(request.method),
+      httpBody: request.body,
+    );
+
+    return webView.loadRequest(urlRequest);
+  }
+
+  @override
+  Future<bool> canGoBack() => webView.canGoBack;
+
+  @override
+  Future<bool> canGoForward() => webView.canGoForward;
+
+  @override
+  Future<void> goBack() => webView.goBack();
+
+  @override
+  Future<void> goForward() => webView.goForward();
+
+  @override
+  Future<void> reload() => webView.reload();
+
+  @override
+  Future<String> evaluateJavascript(String javascript) async {
+    return runJavascriptReturningResult(javascript);
+  }
+
+  @override
+  Future<void> runJavascript(String javascript) async {
+    await webView.evaluateJavaScript(javascript);
```

Comment:
You're going to need [this handling](https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m#L403-L408) in the Dart code, otherwise this method will throw errors all the time.

Reply:
Done.  I added a try catch and added a TODO to ensure the `NSError` is passed back with the `PlatformException`.